workitem/workitem.go

```diff
@@ -18,6 +18,8 @@ type WorkItem struct {
 	Version int
 	// the field values
 	Fields Fields `sql:"type:jsonb"`
+	// the position of workitem
+	Executionorder float64 `sql:"type:jsonb"`
```

Comment:
@aslakknutsen it now is a float64: https://github.com/almighty/almighty-core/pull/804/files#diff-bd043efe71050f0771319a56c8a73754R22